tools/tplgtool2.py

```diff
@@ -840,13 +841,26 @@ def node_name_in_graph(self, node: Container) -> str:
         """
         return self._node_name_in_graph_from_name(node["widget"]["name"])
 
+    def node_sublabel(self, widget: Container) -> str:
+        match widget["widget"]["id"]:
+            case 'PGA':
+                sublabel = f'<BR ALIGN="CENTER"/><SUB>PGA: {widget["kcontrols"][0]["hdr"]["name"]}</SUB>'
```

Comment:
Is `widget["kcontrols"]` never empty?

Reply:
it should be never empty for gain (ipc4) or PGA (ipc3) widget, because there should be (and only be) a mixer control for the widget, or the topology is just wrong. but alsatplg won't help to detect this I guess